esmvalcore/preprocessor/_multimodel.py

```diff
@@ -216,6 +216,10 @@ def _combine(cubes):
             coord.long_name = None
             coord.attributes = None
 
+        for auxcoord in cube.aux_coords:
```

Comment:
```suggestion
        for auxcoord in cube.coords(dimensions=()):
```

To make sure that only scalar coordinates are removed.